config/clusters/leap/common.values.yaml

```
@@ -268,6 +278,13 @@ basehub:
           profile_options:
             image:
               display_name: Image
+              unlisted_choice:
```

```suggestion
              unlisted_choice: *profile_list_unlisted_choice
```

Let's use YAML anchors here to avoid having to put this in two places? This would require we remove the keys here, but I can't figure out how to do that in the GitHub interface